### DISCUSSION TITLE
do you test DWsiam in pysot?

### DISCUSSION THREAD

**Discussion by ghost** [STVIR/pysot#297] [General] [answered]
do you test DWsiam in pysot of VOT2016  and VOT2018?
thanks

**Comment by ZhiyuanChen** [comment]
I'm having troubles understanding your question. 
Did you mean have we tested the dwsiam on VOT2016 and VOT2018?

**Comment by ghost** [comment]
yes




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Zhiyuan Chen"<notifications@github.com&gt;; 
发送时间: 2020年2月22日(星期六) 上午10:52
收件人: "STVIR/pysot"<pysot@noreply.github.com&gt;; 
抄送: "陈柏霖"<2549853330@qq.com&gt;; "Author"<author@noreply.github.com&gt;; 
主题: Re: [STVIR/pysot] do you test DWsiam in pysot? (#297)




I'm having troubles understanding your question.
 Did you mean have we tested the dwsiam on VOT2016 and VOT2018?

**Comment by ZhiyuanChen** [comment, marked as answer]
If you couldn't find it in the README, that means no.
